### PR TITLE
README suggestions

### PR DESCRIPTION
**Pull request by trel**
Is the `irods:::` prefix necessary for `iadmin()`?

The `iauth()` example with and without parameters is not clear, so I didn't touch those.

### CODE REVIEW

**Review by alanking — APPROVED** [review]
Seems good to me. I am averse to advertising the irods_demo for personal reasons but I suppose it's the easiest way to get a REST-surfaced iRODS zone in one line

**Comment by MartinSchobben** [comment]
The `irods:::` is needed as only user facing functions are exported. The function `iadmin()` is not exported and hence requires the `:::`. This only a convenience (helper) function for me to set-up an example for the README file without having to use the actual icommands.

File: README.Rmd [hunk 1]
```diff
@@ -31,43 +31,44 @@ You can install the development version of rirods like so:
...
 
 ```{bash, eval=FALSE}
 # clone the repo
-# git clone https://github.com/irods/irods_demo
-# initiate git submodules
-# git submodule update --init
+# git clone --recursive https://github.com/irods/irods_demo
 # to start
 cd ../irods_demo
 docker-compose up
```
[the remark this answers is not in the document]
Comment (trel) [reply README.Rmd:45]: oh, that's pretty good.

**Comment by trel** [comment]
> The function `iadmin()` is not exported and hence requires the `:::`. 

perfect - makes sense and is succinct.

**Comment by trel** [comment]
tweaked a bit.   i think it will render as intended and tell a helpful story.    can tweak later if not quite right.

will merge upon agreement.

File: README.Rmd [hunk 2]
```diff
@@ -51,7 +51,7 @@ This will result in the REST API running at http://localhost/irods-rest/0.9.3 (o
 
 To connect to the REST API endpoint of your choice, load `rirods`, connect with `create_irods()`, and authenticate with your iRODS credentials:
 
-```{r admin, include=FALSE}
+```{r admin}
```
Comment (MartinSchobben) [inline README.Rmd:54]: Now that I think about this, did we not agree that the r package should be only user facing.

Comment (trel) [reply README.Rmd:54]: Oh, maybe we did?    But if so, then the code cannot prep bobby... and run the docs/demo/tests...

Hmm....

Comment (MartinSchobben) [reply README.Rmd:54]: No, the code is evaluated but not shown. Or do you mean that the example is not entirely obvious from the code shown?

Comment (MartinSchobben) [reply README.Rmd:54]: if, the later we should maybe replace the `iadmin` part with `docker exec` icommands

Comment (trel) [reply README.Rmd:54]: Okay - I think we can leave the `iadmin` functionality there, and like you said, leave it not exported in the package (requiring the  `rirods:::` prefix).

**Pull request merged**